src/lib/plugin.js

```diff
@@ -128,6 +128,17 @@ class FiveBellsLedger extends EventEmitter2 {
     const ledgerMetadata = yield this._fetchLedgerMetadata()
     this.accountUriTemplate = ledgerMetadata.urls.account
 
+    // Set ILP prefix
+    const ledgerPrefix = ledgerMetadata.ilp_prefix
+    this.prefix = this.configPrefix || ledgerMetadata.ilp_prefix
+
+    if (ledgerPrefix && this.configPrefix && ledgerPrefix !== this.configPrefix) {
+      debug('WARNING: ledger prefix does not match local config')
```

Comment:
I would change this to `console.warn`. (Default behavior: visible)

Also would add the configured prefix and the ledger-reported prefix to the message, e.g.

``` js
console.warn('ilp-plugin-bells: ledger prefix does not match locally configured prefix ledger=' + ... + ' local=' + ...)
```